index.bs

```diff
@@ -1726,8 +1726,14 @@ To support obtaining assertions via {{CredentialsContainer/get()|navigator.crede
 {{PublicKeyCredential}}'s [=interface object=]'s implementation of the <dfn for="PublicKeyCredential" method>\[[Create]](origin,
```

Comment:
Could you update the language for [isConditionalMediationAvailable()](https://pr-preview.s3.amazonaws.com/w3c/webauthn/1951/410d0f7...pascoej:2b8521b.html#dom-publickeycredential-isconditionalmediationavailable) to reflect that it only applies to `get` requests?